cmd/gogio/androidbuild.go

```diff
@@ -867,6 +875,15 @@ func getPermissions(ps []string) ([]string, []string) {
 	return permissions, features
 }
 
+func hasForegroundPermission(permissions []string) bool {
```

Comment:
Generalize slightly to

func stringContains(haystack []string, needle string) bool